<?xml version="1.0"?>
<robot name="lightgreen_block">
	<link name="base_link">
		<visual>
			<geometry>
				<mesh filename="package://irsl_sim_environments/urdf/puzzle_blocks/lightgreen_block/lightgreen_block_base_link.dae" />
			</geometry>
		</visual>
		<collision>
			<origin xyz="0.000000000000e+00 0.000000000000e+00 0.000000000000e+00" rpy="0.000000000000e+00 -0.000000000000e+00 0.000000000000e+00" />
			<geometry>
				<box size="3.000000000000e-02 3.000000000000e-02 3.000000000000e-02" />
			</geometry>
		</collision>
		<collision>
			<origin xyz="0.000000000000e+00 0.000000000000e+00 3.000000000000e-02" rpy="0.000000000000e+00 -0.000000000000e+00 0.000000000000e+00" />
			<geometry>
				<box size="3.000000000000e-02 3.000000000000e-02 3.000000000000e-02" />
			</geometry>
		</collision>
		<collision>
			<origin xyz="0.000000000000e+00 -3.000000000000e-02 3.000000000000e-02" rpy="0.000000000000e+00 -0.000000000000e+00 0.000000000000e+00" />
			<geometry>
				<box size="3.000000000000e-02 3.000000000000e-02 3.000000000000e-02" />
			</geometry>
		</collision>
		<inertial>
			<mass value="0.016199999999999999" />
			<origin xyz="0.000000000000e+00 -1.000000000000e-02 2.000000000000e-02" />
			<inertia ixx="8.9099999999999977e-06" ixy="0" ixz="0" iyy="5.6699999999999999e-06" iyz="1.6199999999999993e-06" izz="5.6699999999999991e-06" />
		</inertial>
	</link>
<!-- HotFix for gazebo (manually added) -->
<gazebo reference="base_link">
 <collision>
  <surface>
   <contact>
    <ode>
     <kp>500.0</kp>
     <kd>0.01</kd>
     <max_vel>0.0</max_vel>
     <min_depth>0.0004</min_depth>
    </ode>
   </contact>
   <friction>
    <ode>
     <mu>10.0</mu>
     <mu2>10.0</mu2>
    </ode>
   </friction>
  </surface>
 </collision>
</gazebo>
</robot>
